examples/components/Home.vue

```diff
@@ -5,6 +5,7 @@
       <ul class="examples">
         <li><router-link to="/stars">Stars</router-link></li>
         <li><router-link to="/vuex-example">Vuex Example</router-link></li>
+        <li><router-link to="/wedge2-demo">Field Of Example</router-link></li>
```

Comment:
Should be `/radar-demo` and `Radar Example`

Reply:
Yes right, I renamed wedge2 to radar, and forgot to update here